android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/ssdp/SsdpMessage.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2014 Fraunhofer FOKUS
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * AUTHOR: Louay Bassbouss <louay.bassbouss@fokus.fraunhofer.de>
```

Comment:
Please remove this line.

Instead, please add your name to the CONTRIBUTORS file at:
https://github.com/google/physical-web/blob/master/CONTRIBUTORS